.github/workflows/pr-docu-checks.yml

```diff
@@ -0,0 +1,26 @@
+name: PR Docu Checks
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "release-*"
+  workflow_dispatch:
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Validate links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
https://github.com/kyma-project/kyma-metrics-collector/actions/runs/11142749076

Reply:
well observed, thanks. Changed the action to not use that plugin as we do in telemetry-manager